homeassistant/components/tuya/light.py

```diff
@@ -171,6 +171,14 @@ class TuyaLightEntityDescription(LightEntityDescription):
             entity_category=EntityCategory.CONFIG,
         ),
     ),
+    # Air Purifier
+    # https://developer.tuya.com/en/docs/iot/f?id=K9gf46h2s6dzm
+    "kj": (
+        TuyaLightEntityDescription(
+            key=DPCode.LIGHT,
+            name="Light",
```

Comment:
Are you able to set the entity category?

Reply:
It felt like the entity should show up in the controls section, rather than the configuration or diagnostic section.

Omitting the Category appears to be the only way to achieve this, as "Controls" does not appear to be in the entity category definition

https://github.com/home-assistant/core/blob/ef887e529fd33c7fbf32e19b133072cb8ca466d6/homeassistant/const.py#L1153

Reply:
Ok, I was looking at the 'kt' air conditioner just below.

Based on what you say, this is a more of a functional light than a backlight.  Is that correct or does the 'kt' need changing too?

Reply:
With the Air Purifier i have, its a bit more of a backlight (it turns off the LEDs)
The way it was worded on the Tuya website, it read more like an overall "light".

I'd say its more of a functional light, but neither here nor their about it.

Reply:
I don't have either product or any images, but based on the descriptions it sounds like it should be the same as the backlight below it (name and category).

If not then I recommend adding a comment to the code giving a brief reason for the difference.

This is partly to promote consistency of whatever might be added here in the future.

Reply:
Updated as requested :)